azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/state/LocalUserState.kt

```diff
@@ -53,5 +53,5 @@ internal data class LocalUserState(
     val cameraState: CameraState,
     val audioState: AudioState,
     val videoStreamID: String?,
-    val displayName: String?,
```

Comment:
As we have option to have null display name in composite, thus display name is nullable.

Reply:
addressed.